projects/schematics/src/migrations/2_0/update-cms-component-state_spec.ts

```diff
@@ -175,7 +175,7 @@ const ALL_TEST_CASES_CLASS = `
       StateWithCms,
```

Comment:
I think there is more things to fix in this particular file. I see EntityLoaderState and LoaderState above.

Reply:
Done.